roles/ceph-monitor/files/ceph_adjust.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/python
```

Comment:
Where does target host get the source of this repo?
Will we upload this module to pip repo?